app/blueprints/scoring/templates/macros/scores_justification.html

```diff
@@ -64,11 +64,11 @@ <h2 class="govuk-heading-m">Current score: {{ latest_score['score'] }}</h2>
             <h2 class="govuk-heading-m govuk-!-margin-top-4">Rescore</h2>
             {% endif %}
             <p class="govuk-body">Score this sub criteria against the
-                <a class="govuk-link govuk-link--no-visited-state" href={{ state.fund_guidance_url }}>assessment
+                <a class="govuk-link govuk-link--no-visited-state" href={{ fund_guidance_url }}>assessment
```

Comment:
Really picky but as it's in the round can we call it `round_guidance_url`?

Reply:
Oh good point! Will add

Reply:
Have added in https://github.com/communitiesuk/funding-service-design-assessment/pull/427/commits/b57dc72a6d3add1a4d59c53628d6f90b81120d9b - good spot